code/game/machinery/_machinery.dm

```diff
@@ -377,6 +377,8 @@ Class Procs:
 /obj/machinery/_try_interact(mob/user)
 	if((interaction_flags_machine & INTERACT_MACHINE_WIRES_IF_OPEN) && panel_open && (attempt_wire_interaction(user) == WIRE_INTERACTION_BLOCK))
 		return TRUE
+	if((user.mind?.martial_art) && istype(user.mind.martial_art, /datum/martial_art/buster_style) && (user.a_intent == INTENT_GRAB)) //buster arm shit since it can throw vendors
```

Comment:
```suggestion
	if((user.mind?.has_martialart(MARTIALART_BUSTERSTYLE)) && (user.a_intent == INTENT_GRAB)) //buster arm shit since it can throw vendors
```